lib/version.js

```diff
@@ -1,2 +1,2 @@
 // generated by genversion
-module.exports = '4.1.2-postman.1'
+module.exports = '4.1.2-postman.2'
```

Comment:
This change should not happen in PR. It'll happen when we release a new package.